<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="TotalPayout_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total 
payout</string>
   </property>
  </widget>
  <widget class="QLabel" name="InterestRate_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Interest rate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DepositAmount_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="MonthlyPayment_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Monthly 
payment</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="InterestRate_input">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    border: 1px solid white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="CreditCalcLabel">
   <property name="geometry">
    <rect>
     <x>172</x>
     <y>20</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Credit</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Clear_Credit">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear all input</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate_Credit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>381</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="Term_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Term</string>
   </property>
  </widget>
  <widget class="QLabel" name="Overpayment_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Overpayment</string>
   </property>
  </widget>
  <widget class="QLabel" name="DepositAmount_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deposit 
Amount</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_m_payment">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_overpayment">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_payout">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>440</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="RUB">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>RUB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="Percent">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>%</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="Term_comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
border: 1px solid white;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="Type_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Type_comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 1px solid white;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
